### DISCUSSION TITLE
2006/10/27/labview-%E4%BB%A3%E7%A0%81%E4%B8%AD%E5%B8%B8%E8%A7%81%E7%9A%84%E9%94%99%E8%AF%AF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#125] [Announcements]
# 2006/10/27/labview-%E4%BB%A3%E7%A0%81%E4%B8%AD%E5%B8%B8%E8%A7%81%E7%9A%84%E9%94%99%E8%AF%AF

发现了程序的问题再回头去调试，在查找程序错误时就不可避免地要花大量时间。要调高开发效率，最好是在编写代码时就避免一些常见的低级错误，这样可以节约大量的调试时间。 有些编程错误差不多是每个 LabVIEW 程序员都曾遇到过的。在编写相关代码的时候，对这些问题多留心一下，就可以大大减少调试时间。

https://blog.qizhen.xyz/2006/10/27/labview-%E4%BB%A3%E7%A0%81%E4%B8%AD%E5%B8%B8%E8%A7%81%E7%9A%84%E9%94%99%E8%AF%AF

**Comment by ruanqizhen** [comment]
之前博客下的留言：


youju
2006年10月21日 21:19 编辑
LABVIEW是啥东西已全无记忆，，记得以前我用的是SYSTEMVIEW来模拟的。。。
